.github/workflows/amplify_core.yaml

```diff
@@ -0,0 +1,47 @@
+name: amplify_core
+on:
+  push:
+    branches:
+      - main
+      - stable
+      - next
+  pull_request:
+    paths:
+      - 'packages/amplify_core/**/*.dart'
```

Comment:
Fore these patterns, I dk if would make more sense to specify what should NOT run the tests, like `!**/*README.md` than trying to state what SHOULD and leave this open to errors of omission, but I do not feel strongly one way or the other.

Reply:
My feeling is that CI checks in PRs should do the absolute minimum amount of work to verify the change. This is best accomplished through an "opt-in" rule vs "opt-out". On push, CI checks run for _any_ change and this is a better mechanism for catching things which weren't immediately obvious.